### DISCUSSION TITLE
Violation - Families Policy Requirements: Families Data Practices

### DISCUSSION THREAD

**Discussion by hawkeyed** [admob-plus/admob-plus#658] [Q&A]
"Details
We’ve identified that your app or an SDK in your app transmits device identifier(s) from children or users of unknown age that do not comply with our [Families Policy](https://support.google.com/googleplay/android-developer/answer/9893335#families-policy).
These identifiers may include but are not limited to Advertising ID."
I have this problem. Is there any solution for this?

**Comment by EMI-INDO** [comment]
@hawkeyed 

`<uses-permission android:name="com.google.android.gms.permission.AD_ID" tools:node="remove"/>`

<img width="1420" alt="remove-ad-id" src="https://github.com/admob-plus/admob-plus/assets/78555833/037184db-b873-4978-8f39-3f5ad5e38479">

**Reply by hawkeyed** [reply]
Thank you but I tried this before. I got build error as far as I remember. Probably I didn't have that xmlns:tools line. I solved the issue with changing target age to 13+ and uploaded a new version. Age is still 3+ in play store. I just opt out of families policy programme.
